keras_cv/models/convmixer.py

```diff
@@ -112,29 +112,23 @@ def apply(x):
     return apply
 
 
-class PatchEmbed(layers.Layer):
+def PatchEmbed(dim, patch_size):
     """Implementation for Extracting Patch Embeddings.
     Args:
-        dim: integer, Number of filters for convolution layers.
+        inputs: Input tensor.
```

Comment:
It looks like this is now incorrect. Other than this, LGTM

Reply:
Done!